src/platform-implementation-js/dom-driver/gmail/views/gmail-collapsible-section-view.module.css

```diff
@@ -0,0 +1,16 @@
+.title {
+  color: rgb(0 0 0 / 54%) !important;
+}
+
+.subtitle {
+  margin: 0;
+  color: rgb(0 0 0 / 87%) !important;
```

Comment:
is there a way to use specificity instead of `!important` here? Seems like it might cause the user to have a worse time if they have to use `!important` and/or not have their styling work?

Reply:
The trickiness here is that for title and subtitle, Gmail has `.Wr>.Wn` as the ruleset selector setting color. For the CSS local `.subtitle`, specifically, we could do something like `.title .subtitle`. For `.title` though, we would need to pick a parent.

Reply:
Yup, we've used the same title parent selector for 5 years though, so maybe that's safe as a parent to use here too.

Reply:
See [ac037be](https://github.com/InboxSDK/InboxSDK/pull/1118/commits/ac037be673306d5d8891c08f714910e02e41fa94).